src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -585,7 +585,7 @@ Pinetime::Controllers::Settings::Colors WatchFacePineTimeStyle::GetPrevious(Pine
   if (colorAsInt > 0) {
     prevColor = static_cast<Controllers::Settings::Colors>(colorAsInt - 1);
   } else {
-    prevColor = static_cast<Controllers::Settings::Colors>(16);
+    prevColor = static_cast<Controllers::Settings::Colors>(17);
   }
   return prevColor;
-}
+}
```

Comment:
Please don't remove the last empty line at the end of the files. Try to keep the commits clean.

```suggestion
}

```